src/circuit/circuit_frame.rs

```diff
@@ -4624,7 +4664,8 @@ fn apply_continuation<F: LurkField, CS: ConstraintSystem<F>>(
 
     // This is all clunky because we can't currently return AllocatedBit from case expressions.
     let newer_cont2_not_dummy_result_num = case_results[7].clone();
-    let newer_cont2_not_dummy_ = equal!(cs, &newer_cont2_not_dummy_result_num, &g.true_num)?;
+    // let newer_cont2_not_dummy_ = equal!(cs, &newer_cont2_not_dummy_result_num, &g.true_num)?;
```

Comment:
Is the commented line of code serving us better than sentence on what we'd ideally like to see here?
Also, do we have an issue for potential elisions of `g.true_num`? We have a few:
```
huitseeker@lightning.local➜~/tmp/lurk-rs(optimize-symbol-comparisons✗)» rg 'g\.true_num'                                                                                                                                           [12:02:02]
src/circuit/gadgets/case.rs
241:        let mut acc = g.true_num.clone();

src/circuit/circuit_frame.rs
687:        results.add_clauses_expr(ExprTag::Nil, expr, env, cont, &g.true_num);
688:        results.add_clauses_expr(ExprTag::Num, expr, env, cont, &g.true_num);
689:        results.add_clauses_expr(ExprTag::Fun, expr, env, cont, &g.true_num);
690:        results.add_clauses_expr(ExprTag::Char, expr, env, cont, &g.true_num);
691:        results.add_clauses_expr(ExprTag::Str, expr, env, cont, &g.true_num);
692:        results.add_clauses_expr(ExprTag::Comm, expr, env, cont, &g.true_num);
693:        results.add_clauses_expr(ExprTag::Key, expr, env, cont, &g.true_num);
694:        results.add_clauses_expr(ExprTag::U64, expr, env, cont, &g.true_num);
726:            &g.true_num,
1691:        &g.true_num,
1713:    results.add_clauses_cons(*p.quote.value(), &arg1_or_expr, env, &the_cont, &g.true_num);
2316:        &g.true_num,
3036:        &g.true_num,
3912:            &g.true_num,
4360:        &g.true_num,
4374:        &g.true_num,
4667:    // let newer_cont2_not_dummy_ = equal!(cs, &newer_cont2_not_dummy_result_num, &g.true_num)?;
4674:        &g.true_num,
4792:        &g.true_num,
4798:        &g.true_num,
4805:        &g.true_num,
4811:        &g.true_num,
```

Reply:
The commented line was just an oversight and should have been removed. Done now.